internal/helper/yawollet.go

```diff
@@ -344,17 +356,25 @@ func createEnvoyTCPListener(
 			},
 		},
 		FilterChains: []*envoylistener.FilterChain{{
-			// proxy filter has to be the last in the chain
-			Filters: append(filters, &envoylistener.Filter{
-				Name: envoywellknown.TCPProxy,
-				ConfigType: &envoylistener.Filter_TypedConfig{
-					TypedConfig: listenPort,
+			Filters: append(filters,
+				&envoylistener.Filter{
+					// NOTE: this is not present in envoywellknown
+					Name: "extensions.filters.network.connection_limit",
```

Comment:
There is a related constant not mentioned in `envoywellknown`. Maybe you want to put them closed together so you can easily maintain those constants when `envoywellknown` starts adding them.